### PR TITLE
feature: add tc are suggestions in exec

### PR DESCRIPTION
**Pull request by GregoryEssertel**
https://github.com/strateos/lab/pull/1496

```
$ transcriptic preview Seal | transcriptic exec --api localhost:9000 --no-redirect -w wc0-mcx1
Sending request to localhost:9000
Success. View localhost:9000/dashboard to see the scheduling outcome.

$ transcriptic preview Seal | transcriptic exec --api localhost:9000 --no-redirect -w wc0-mcx1 -tc-suggestion
Sending request to localhost:9000
Success. View localhost:9000/dashboard to see the scheduling outcome.

$ transcriptic preview Seal | transcriptic exec --api localhost:9000 -w wc0-mcx1 -tc-suggestion
Invalid api target, expects http://base/facility/workcell.
```

### CODE REVIEW

File: transcriptic/cli.py [hunk 1]
```diff
@@ -672,6 +672,12 @@ def format_cmd(manifest):
     default=None,
     help="Delay in minutes at which the given protocol should start (at the earliest).",
 )
+@click.option(
+    "--time-constraints-are-suggestion",
+    "-tc-suggestion",
+    help="If set, the time constraints will be considered only suggestion.",
+    is_flag=True,
+)
```
Comment (EribertoLopez) [inline transcriptic/cli.py:680]: I think we should include the default value you set in this [PR](https://github.com/strateos/lab/pull/1496/files#diff-515485d184b82bc0157ab6ea7b86ea2422e5cfb280b59334b881a99d05b80490R83-R88)

Comment (GregoryEssertel) [reply transcriptic/cli.py:680]: Like it is a flag, I think the value will be false if the flag is not set, or true if it is set. But I haven't done a full pipeline test, I will do.

Comment (scottyantipa) [reply transcriptic/cli.py:680]: should be "only a suggestion"

**Review by scottyantipa — DISMISSED** [review]

File: transcriptic/cli.py (hunk 1 above)
Comment (EribertoLopez) [reply transcriptic/cli.py:680]: Got it, makes sense from the comments on the other PR.

**Comment by GregoryEssertel** [comment]
@scottyantipa @EribertoLopez I did add a new flag to be able to use local dev and give the node address directly.
That is an extra change!

**Review by EribertoLopez — APPROVED** [review]

**Pull request merged**